sdk/web-pubsub/web-pubsub-express/samples-dev/server.ts

```diff
@@ -12,16 +12,19 @@ const handler = new WebPubSubEventHandler("chat", ["https://xxx.webpubsub.azure.
   dumpRequest: false,
   handleConnect(req, res) {
     console.log(req);
-    res.success();
+    // You can set the state for the connection, it lasts throughout the lifetime of the connection
+    res.setState("calledTime", 1).success();
```

Comment:
I'm torn about the chaining/fluent style here. @bterlson what do you think?

Reply:
any design suggestions?

Reply:
The alternative is mostly to just not do it, e.g. have `setState` return void and the above code would instead be:

```ts
res.setState("calledTime", 1);
res.success();
```

We usually don't do fluent patterns anymore, but I'm not sure if we actively are discouraging them.

Reply:
Follow expert's pattern then